pkg/apis/triggers/v1alpha1/param.go

```diff
@@ -0,0 +1,24 @@
+package v1alpha1
+
+// ParamSpec defines arbitrary parameters needed beyond typed inputs (such as
+// resources). Parameter values are provided by users as inputs on a TaskRun
+// or PipelineRun.
+type ParamSpec struct {
+	// Name declares the name by which a parameter is referenced.
+	Name string `json:"name"`
+	// Description is a user-facing description of the parameter that may be
+	// used to populate a UI.
+	// +optional
+	Description string `json:"description,omitempty"`
+	// Default is the value a parameter takes if no input value is supplied. If
+	// default is set, a Task may be executed without a supplied value for the
+	// parameter.
+	// +optional
+	Default *string `json:"default,omitempty"`
```

Comment:
typically we'd use a pointer for types that are optional so that their absence can be indicated with the nil value. We definitely use that  property for default. That being said, it beats me why Description is no also a pointer 🤔 I'm inclined to keep it this way so that we don't break anyone (e.g. CLI) by changing the behavior when no description is provided from "" to nil.

Reply:
It is a pointer because a `Default` value could be an empty string (from what I remember)